Zappy_IA_Report/DTC/Video/Tutorial2.py

```
@@ -0,0 +1,29 @@
+import pandas as pd
+import numpy as np
+
+df = pd.read_csv('500hits.csv', encoding = 'latin-1')
```

Flake8 (7.2.0)

[error] unexpected spaces around keyword / parameter equals

(E251)